.github/workflows/branch.yml

```diff
@@ -0,0 +1,15 @@
+name: Branch CI
+
+on:
+  push:
+    branches-ignore:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docker:
+    name: 'Run docker image building'
+    uses: ./.github/workflows/docker.yml
```

Comment:
what is with the `ci.yml` workflow? it also builds the docker image right? do we have it twice now?

Reply:
maybe we should drop the PR building then since we do the branch anyways?

Reply:
well... we can always remove that afterwards... better to have too many than too less ^^